### ISSUE TITLE
Change easing function of the image slider

### ISSUE THREAD

**Issue by savicaleksa** [igordanchenko/yet-another-react-lightbox#80]
### Is your feature request related to a problem? Please describe.

I haven't been able to find a way to change the easing function of the sliding animation, which seems to be linear. 

### Describe the solution you'd like

It would be nice if there was an `easing` prop for the `Lightbox` component, which would support different easing functions (ease, ease-in, ease-out, etc.) so we wouldn't have to stick with linear animations.

**Comment by igordanchenko** [comment]
YARL currently supports only linear easing for two reasons:

1) ease, ease-in, and ease-in-out do not work well for swipe animation as the swipe animation looks jerky. When the lightbox picks up the swipe gesture (either touch swipe on mobile or touchpad swipe on laptop), it must start playing the animation at the same pace as the image was moving right before the animation kicked in.

2) I didn't particularly like how ease, ease-out, and ease-in-out worked with the Thumbnails plugin. All of these easing options made an impression that the main image and the thumbnails were moving out of sync, and the linear easing was the least distracting compromise.

If you'd like to see what I'm talking about, here is a patch that adds `animation.easing` prop. 

https://gist.github.com/igordanchenko/2c991613a82afa59a6babf6f1f1789e9

If you get a chance, please try out this patch-package and let me know which easing option looks visually appealing to you.

**Comment by savicaleksa** [comment]
Is there a way to install that patch via npm or yarn? I want to try it, and possibly use it in my project.
Thank you very much!

**Comment by igordanchenko** [comment]
Yes, of course. Save it in the `patches` folder, and follow the [`patch-package`](https://www.npmjs.com/package/patch-package) installation instructions.

**Comment by savicaleksa** [comment]
This patch works perfectly in my opinion, thank you for adding a way to do this for those that really need it.

**Comment by igordanchenko** [comment]
Just out of curiosity, what easing setting did you end up using?

**Comment by savicaleksa** [comment]
I've tried a few easing functions and ended up using the built in "ease-out" function. It looked better with the rest of my project because all animated elements had either that, or something very similar. The way default linear easing snapped at the end is something I wanted to change, and I did thanks to this patch.

**Comment by igordanchenko** [comment]
Thank you for the feedback. I'm going to release this as a feature in the upcoming release. While the above patch is fine as a short-term solution, it now makes sense to add this as an officially supported feature in the library.

**Comment by savicaleksa** [comment]
Sounds great! This is already a great start. I forgot to mention that I did have to add `@ts-ignore` because the type definitions haven't been patched yet, so you might want to patch that as well.

**Comment by igordanchenko** [comment]
Actually type definitions were updated as part of the above patch, and TS did pick them up in my test. Maybe you need to restart your TS or IDE.

**Comment by savicaleksa** [comment]
You're right, I just had to restart VS Code and now everything works.
Sorry for the late reply, had some problems with the ISP.

**Comment by igordanchenko** [comment]
This feature is now available in v2.6.0

```tsx
animation={{
  swipe: { easing: "ease-out" },
  navigation: { easing: "ease-in-out" },
}}
```
